### ISSUE TITLE
Create rule S6849 (`jsx-a11y/html-has-lang`) and (`jsx-a11y/lang`): HTML elements should have a valid language property

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4382]
[html-has-lang plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/html-has-lang.md#jsx-a11yhtml-has-lang)

[lang plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/lang.md#jsx-a11ylang)

[RSPEC PR](https://github.com/SonarSource/rspec/pull/3439)
